store-implementation/proxy-store/src/test/resources/mock-proxy-store.properties

```diff
@@ -17,4 +17,4 @@ gaffer.store.class=uk.gov.gchq.gaffer.proxystore.SingleUseMapProxyStore
 gaffer.store.properties.class=uk.gov.gchq.gaffer.proxystore.ProxyProperties
 gaffer.host=localhost
 gaffer.port=8080
-gaffer.context-root=/rest/v1
+gaffer.context-root=/rest/v2
```

Comment:
I'm assuming the proxy store won't work if given /rest/v1? If it now requires a rest api version 2 then we will need to change this context root to just /rest and automatically append v2 to the end.

Reply:
Done.